utilities/ConvertTo-PoshTools.ps1

```diff
@@ -97,7 +97,8 @@ function Out-ObjectProperty {
             $value =  "New-Object -TypeName '$($PropertyType.FullName)' -ArgumentList @('$($parts[0])', $($parts[1].Trim('pt')))"
         }
         elseif ($propertyType.FullName -eq "System.Drawing.Color") {
-            $value =  "[System.Drawing.Color]::FromArgb($($property.InnerText))"
+            $color =  "[System.Drawing.Color]::$($property.InnerText)"
```

Comment:
This will return an array of ints if the color is a custom color so we would probably need a way to figure that out.